docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -158,66 +166,108 @@ The list of properties that are currently exposed via this method are:
 
 ### Features
 
-This is being implemented, so it's not currently available.
+So far, this function is only enabled for the `global` section in `experimental-theme.json`.
```

Comment:
What if we add an intro that sums up the general behavior ([see](https://github.com/WordPress/gutenberg/pull/22291#pullrequestreview-418254189)). I'm thinking something along these lines:

---

Via the `features` key we allow controlling some editor and block features: 

- The features within `global` will override the default editor settings, which plugins still can modify by hooking into the `block_editor_settings` filter in the server.
- The features within each block content will override the default block settings, which plugins can still modify by hooking into the `blocks.registerBlockType` filter in the client.

Block settings take precedence over global settings.

For example, if a feature is enabled for a block by default, although the theme disables it at the global level, the block will still have it enabled. If the theme wants to override a block's default, it has to target that particular block.

Reply:
Great idea, how do you like e7ada48?